### PR TITLE
Add some tests to JapaneseCalendar

### PR DESCRIPTION
**Pull request by hughbe**
/cc @tarekgh

### CODE REVIEW

File: src/System.Globalization.Calendars/tests/JapaneseCalendar/JapaneseCalendarToFourDigitYear.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+{
+    public class JapaneseCalendarToFourDigitYear
+    {        
+        [Theory]
+        [InlineData(1)]
+        [InlineData(99)]
+        [InlineData(2016)]
+        public void ToFourDigitYear(int year)
+        {
+            Calendar calendar = new JapaneseCalendar();
```
Comment (tarekgh) [inline src/System.Globalization.Calendars/tests/JapaneseCalendar/JapaneseCalendarToFourDigitYear.cs]: I suggest you set explicit TwoDigitYearMax value before checking the ToFourDigitYear. doing that will guarantee the result regardless of the running OS.

Comment (hughbe) [inline src/System.Globalization.Calendars/tests/JapaneseCalendar/JapaneseCalendarToFourDigitYear.cs]: done

**Comment by tarekgh** [comment]
LGTM

**Comment by stephentoub** [comment]
LGTM2

**Pull request merged**